tests/e2e_test/manifest_reconciliation_test.go

```diff
@@ -0,0 +1,115 @@
+	kyma := NewKymaWithSyncLabel("kyma-sample", "kcp-system", v1beta2.DefaultChannel,
+		v1beta2.SyncStrategyLocalSecret)
+	module := NewTemplateOperator(v1beta2.DefaultChannel)
+
+	InitEmptyKymaBeforeAll(kyma)
+	CleanupKymaAfterAll(kyma)
+
+	Context("Given kyma deployed in KCP", func() {
+		It("When enabling Template Operator", func() {
+			Eventually(EnableModule).
+				WithContext(ctx).
+				WithArguments(runtimeClient, defaultRemoteKymaName, remoteNamespace, module).
+				Should(Succeed())
+			By("Then the Module Operator is deployed on the SKR cluster")
+			Eventually(CheckIfExists).
+				WithContext(ctx).
+				WithArguments("template-operator-controller-manager", "template-operator-system", "apps", "v1",
+					"Deployment", runtimeClient).
+				Should(Succeed())
+			By("And the SKR Module Default CR is in a \"Ready\" State")
+			Eventually(CheckSampleCRIsInState).
+				WithContext(ctx).
+				WithArguments("sample-yaml", "kyma-system", runtimeClient, "Ready").
+				Should(Succeed())
+			By("And the KCP Kyma CR is in a \"Ready\" State")
+			Eventually(KymaIsInState).
+				WithContext(ctx).
+				WithArguments(kyma.GetName(), kyma.GetNamespace(), controlPlaneClient, shared.StateReady).
+				Should(Succeed())
+		})
+
+		It("When the Manifest is labelled to skip reconciliation", func() {
+			labels, err := GetManifestLabels(ctx, kyma.GetName(), kyma.GetNamespace(), module.Name, controlPlaneClient)
+			Expect(err).ToNot(HaveOccurred())
+			labels[declarativev2.SkipReconcileLabel] = "true"
+			err = SetManifestLabels(ctx, kyma.GetName(), kyma.GetNamespace(), module.Name, controlPlaneClient, labels)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("When deleting the SKR Default CR")
+			/*err = SetFinalizer("sample-yaml", "kyma-system", "operator.kyma-project.io",
+				"v1alpha1", "Sample", []string{}, runtimeClient)
+			Expect(err).ToNot(HaveOccurred())*/
+			err = DeleteCRWithGVK(ctx, runtimeClient, "sample-yaml", "kyma-system",
+				"operator.kyma-project.io", "v1alpha1", "Sample")
+			Expect(err).ToNot(HaveOccurred())
+			By("Then SKR Module Default CR is not recreated")
+			Consistently(CheckIfNotExists).
+				WithContext(ctx).
+				WithArguments("sample-yaml", "kyma-system", "operator.kyma-project.io",
+					"v1alpha1", "Sample", runtimeClient).
+				Should(Succeed())
+
+			By("When deleting the SKR Module Manager Deployment")
+			err = DeleteCRWithGVK(ctx, runtimeClient, "template-operator-controller-manager",
+				"template-operator-system", "apps", "v1", "Deployment")
+			Expect(err).ToNot(HaveOccurred())
+			By("Then Module Manager Deployment is not recreated on the SKR cluster")
+			Consistently(CheckIfNotExists).
+				WithContext(ctx).
+				WithArguments("template-operator-controller-manager", "template-operator-system",
+					"apps", "v1", "Deployment", runtimeClient).
+				Should(Succeed())
+		})
+
+		It("When the Manifest skip reconciliation label removed",
+			func() {
```

Comment:
is it really necessary to make this `func()` here in a separate line? it's regarding the indenting of function body alignment, this whole function body is not aligned with other function which affect the format of the code.

Can you reformate it?